message_sender/tests.py

```diff
@@ -4161,6 +4161,7 @@ def test_send_hsm(self):
             json.loads(request.body),
             {
                 "to": "27820001001",
+                "ttl": 604800,
```

Comment:
We only really want this here if the `MESSAGE_TTL` is set on the channel config. If it isn't set, then it shouldn't send a `ttl` with the message.

Reply:
how would it be included in the tests then?

Reply:
For this specific test, at the beginning you see we're creating the `sender` with a bunch of arguments. We need to add the TTL to that list of arguments, similar to how the factory does it when it creates the sender.